ompi/Makefile.am

```diff
@@ -172,6 +175,8 @@ ompidir = $(ompiincludedir)/$(subdir)
 nobase_ompi_HEADERS = $(headers)
 endif
 
+include $(top_srcdir)/Makefile.ompi-rules
```

Comment:
why?

Reply:
I think I mentioned this one verbally on the phone: I changed the build scheme to traverse into the two man page directories so that it is easier for a developer writing man pages to iterate over writing a man page and checking how it rendered.

Specifically, when there is a `Makefile.am` in the man page directories, you can just invoke `make` in an individual directory and see how the page you are working on rendered.  Otherwise, you have to invoke `make` at the top-level `ompi` dir, which has to traverse a bunch of directories before it renders the page you are working on.

Put simply: it's an optimization for those who are working on man pages.  I figured the trivial time that it adds to the general build was worth it.

Reply:
Wait; I read Brian's comment wrong / out of context.

I think that this `include` is left over from when I was experimenting with a few different ways to get the makefile rules in the right places.  I'll double check, but I think this new `include` here in `ompi/Makefile.am` should be deleted.

Reply:
Yes, I was just asking about the include of the Makefile.ompi-rules, which I don't think is needed when the manpages are split out.